common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -682,6 +682,7 @@
 
                         state.el.trigger('language_menu:change', [langCode]);
                         self.fetchCaption();
+                        self.closeLanguageMenu(e);
```

Comment:
The language menu and speed menus behaved differently before. The speed menu would close when a selection was made but the language menu wouldn't. I wanted both menus to behave the same so I needed to make the language menu close when a choice was selected.